app/models/redirect.rb

```diff
@@ -50,20 +50,20 @@ def paths
   end
 
   def build_url_for path
-    URI.parse("http://" + strip_protocol_from_path(path))
```

Comment:
I am not exactly clear why we were doing `strip_protocol_from_path` in the first place. It seems like it just removed any `http` or `https` and then just immediately replaced it with an `http`...

Reply:
The only reason I saw was because the regex in `add_leading_slash` method was only handling `http`. Now it handle both `http` and `https`. 

So I refactor this because : 

1. The method seems to be pretty useless now
2. It remove to possibility to redirect to a secure link

Maybe  @veganstraightedge  can explain us if there any other reasons to strip the protocol and always keep it to `http`.

Reply:
Feel free to tear this all down and re-build it if yall wanna. My implementation was very quick and very dirty. I'm sure it can be done in a better way.

Reply:
@veganstraightedge @just1602 in that case, I say just ship this. there is no real need to refactor right now (the code is not coupled to anything else, it is not blocking other features, and there haven't rally been any bugs besides this one)

I would prefer to just leave it be and let the test coverage continue to build up around any other edge cases that may come up. That way, if we *do* refactor we can do it with confidence.

Also, right now 2 people are pretty familiar with this code (@just1602 + @veganstraightedge), after a refactor it would be back down to 1